.github/workflows/build.yaml

```diff
@@ -311,6 +314,7 @@ jobs:
           version: ${{ needs.build.outputs.viewer_version }}
 
   post-mac-symbols:
+    if: ${{ secrets.BUGSPLAT_USER && secrets.BUGSPLAT_PASS }}
```

Comment:
This may be not enough, as we have the following error on a mac build step:
`=== FAILURE BUGSPLAT_USER or BUGSPLAT_PASS missing, and no $build_secrets_checkout/bugsplat/bugsplat.sh`
([example](https://github.com/secondlife/viewer/actions/runs/8157661128/job/22299169482))